README.md

```diff
@@ -1 +1,81 @@
-# book-store
\ No newline at end of file
+# Math Magicians
+
```

Comment:
- [ ] [optional] I would suggest using the correct name for the project, It will make your README file looks more professional.